frontend/components/Errors/BaseError.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from "../Button/Button"
+import { Headline } from "../Headline/Headline"
+
+type BaseErrorProps = {
+    /** The title of the page */
+    headline: string
+    /** Custom descripton for example can be just a little text for more information or a list with the reasons. */
+    description: React.ReactNode
+}
+
+/**
+ * Base Error Component, is used as a wrapper for errors like 404, 500,...
+ */
+export const BaseError: React.FC<BaseErrorProps> = ({ headline, description }) => {
+    return <div className="container mt-10 md:mt-40">
```

Comment:
```suggestion
    return <div className="container">
```

mt wegmachen das sieht bei macbooks (bzw. langen monitoren kacke aus) auserdem ist das web nun mal von oben nach unten ... wenn dann mit screen height arbeiten aber sowas nicht: 

![image](https://user-images.githubusercontent.com/47084064/204010342-4d6e0dbc-7020-41cf-b0c8-476fe4fc2dc1.png)